Changelog.md

```diff
@@ -1,27 +1,142 @@
-# 1.1.26 -- 2018-07-06
```

Comment:
In addition to the change to `jsonErrors`, I had to merge `changelog.md` and `Changelog.md` before doing anything else, because on my OS, those two paths are considered to point to the same file, so git was always complaining that at least one of the two files was different from the repo.